From: virtudude <virtudude@protonmail.com>

--- a/arch/arm64/boot/dts/qcom/qcs8550-retroidpocket-rp6.dts
+++ b/arch/arm64/boot/dts/qcom/qcs8550-retroidpocket-rp6.dts
@@ -8,7 +8,7 @@
 #include "qcs8550-ayn-common.dtsi"
 
 / {
-	model = "Retroid Pocket 6";
+	model = "Retroid Pocket 6 (D-Pad Bottom)";
 	compatible = "retroidpocket,rp6", "qcom,qcs8550", "qcom,sm8550";
 	qcom,board-id = <0x1001f 0x00>;
 	qcom,msm-id = <0x25b 0x20000>;
